routes/baseActionsRouter.js

```diff
@@ -119,64 +101,50 @@ router.get("/", function(req, res, next) {
 			}
 		}
 
-		if (getblockchaininfo.chain !== 'regtest') {
-			promises.push(coreApi.getChainTxStats(getblockchaininfo.blocks - 1));
+		if (data.blockChainInfo.chain !== 'regtest') {
+			promises.push(coreApi.getChainTxStats(data.blockChainInfo.blocks - 1));
 		}
 
-		coreApi.getBlocksByHeight(blockHeights).then(function(latestBlocks) {
-			res.locals.latestBlocks = latestBlocks;
-			res.locals.blocksUntilDifficultyAdjustment = ((res.locals.difficultyPeriod + 1) * coinConfig.difficultyAdjustmentBlockCount) - latestBlocks[0].height;
-
+		res.locals.blocksUntilDifficultyAdjustment = ((res.locals.difficultyPeriod + 1) * coinConfig.difficultyAdjustmentBlockCount) - data.blockList[0].height;
 
-			Promise.all(promises).then(function(promiseResults) {
-				res.locals.mempoolInfo = promiseResults[0];
-				res.locals.miningInfo = promiseResults[1];
+		Promise.all(promises).then(function(promiseResults) {
+			res.locals.mempoolInfo = promiseResults[0];
+			res.locals.miningInfo = promiseResults[1];
 
-				var rawSmartFeeEstimates = promiseResults[2];
+			var rawSmartFeeEstimates = promiseResults[2];
 
-				var smartFeeEstimates = {};
+			var smartFeeEstimates = {};
```

Comment:
wonder if while at it we could also remove this deadcode

Reply:
good idea, i'll update it soon

Reply:
code removed